4_setup_train_ff_topologies/2025_09_11_uci_full_split/log.txt

```diff
@@ -0,0 +1,379 @@
+LICENSE: Could not open license file "oe_license.txt" in local directory
+LICENSE: N.B. OE_LICENSE environment variable is not set
+LICENSE: N.B. OE_DIR environment variable is not set
+LICENSE: No product keys!
+LICENSE: No product keys!
+LICENSE: No product keys!
+LICENSE: No product keys!
+The OpenEye Toolkits are found to be installed but not licensed and therefore will not be used.
+The OpenEye Toolkits require a (free for academics) license, see https://docs.eyesopen.com/toolkits/python/quickstart-python/license.html
+The OpenEye Toolkits are found to be installed but not licensed and therefore will not be used.
+The OpenEye Toolkits require a (free for academics) license, see https://docs.eyesopen.com/toolkits/python/quickstart-python/license.html
+2025-09-11 17:32:02.651 | INFO     | __main__:main:546 - Loading dataset: /dfs9/dmobley-lab/jclark9/descent/back-to-school-jen/3_split_train_test/full_split_uci/data-train
+2025-09-11 17:32:02.651 | INFO     | __main__:load_dataset:488 - Loading dataset: /dfs9/dmobley-lab/jclark9/descent/back-to-school-jen/3_split_train_test/full_split_uci/data-train
+/data/homezvol2/jclark9/micromamba/envs/descent-workflow/lib/python3.12/site-packages/datasets/table.py:1421: FutureWarning: promote has been superseded by promote_options='default'.
+  table = cls._concat_blocks(blocks, axis=0)
+2025-09-11 17:32:09.323 | INFO     | __main__:main:550 - Processing all 23109 molecules
+2025-09-11 17:32:09.324 | INFO     | __main__:prepare_to_train:233 - Loading force field: /dfs9/dmobley-lab/jclark9/descent/openff-2.2.1.offxml
+2025-09-11 17:32:09.549 | INFO     | __main__:validate_molecular_dataset:92 - Validating molecules...
+2025-09-11 17:33:24.821 | INFO     | __main__:validate_molecular_dataset:120 - All molecules passed validation
+2025-09-11 17:33:24.822 | INFO     | __main__:prepare_to_train:241 - Processing 23109 molecules.
+2025-09-11 17:33:29.626 | INFO     | __main__:prepare_to_train:249 - Found 32 workers for interchange creation
+
Creating Interchanges:   0%|          | 0/23109 [00:00<?, ?it/s]LICENSE: Could not open license file "oe_license.txt" in local directory
```

Comment:
It's not the end of the world, but I'd really recommend using OpenEye ELF10 charges here

Reply:
I'll run with OpenEye and Sage 2.2.1 after the HPC comes back online